### PR TITLE
Update EVM RPC documentation

### PR DESCRIPTION
**Pull request by rvanasa**
Documentation update for the Tritium release of the EVM RPC canister.

### CODE REVIEW

**Review by dfx-json — APPROVED** [review]

**Review by jessiemongeon1 — APPROVED** [review]

File: docs/developer-docs/multi-chain/ethereum/evm-rpc/costs.mdx [hunk 1]
```diff
@@ -37,7 +37,7 @@ Here is an approximate cost breakdown in USD for an RPC request on the Ethereum
 
 - Total cost: ~ $0.0021 (assuming consensus between 3 RPC providers)
 
-  - Cost per RPC provider: ~ $0.000686 (assuming 1kb request and 1kb response)
+  - Cost per RPC provider: ~ $0.000686 (assuming 1kB request and 1kB response)
```
Comment (THLO) [inline docs/developer-docs/multi-chain/ethereum/evm-rpc/costs.mdx:40]: Shouldn't it be 1KiB?

**Review by THLO — APPROVED** [review]

File: docs/developer-docs/multi-chain/ethereum/evm-rpc/costs.mdx (hunk 1 above)
Comment (jessiemongeon1) [inline docs/developer-docs/multi-chain/ethereum/evm-rpc/costs.mdx:40]:
```suggestion
  - Cost per RPC provider: ~ $0.000686 (assuming 1KiB request and 1KiB response)
```

Comment (rvanasa) [reply docs/developer-docs/multi-chain/ethereum/evm-rpc/costs.mdx:40]: The calculation was based on 1000 request / response bytes, which should be "kB" if I remember correctly (where KiB is 1024 bytes)?

Comment (jessiemongeon1) [reply docs/developer-docs/multi-chain/ethereum/evm-rpc/costs.mdx:40]: Got it, the current version is correct then.

**Comment by rvanasa** [comment]
Merging since the new version is released; we can continue making changes in a follow-up PR if needed.

**Pull request merged**